src/main/src/main/java/org/geoserver/catalog/WMTSStoreInfo.java

```diff
@@ -0,0 +1,108 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.catalog;
+
+import java.io.IOException;
+
+import org.geotools.data.wmts.WebMapTileServer;
+import org.opengis.util.ProgressListener;
+
+/**
+ * A store backed by a {@link WebMapServer}, allows for WMS cascading
```

Comment:
Not WMS, but WMTS. I'm not the author :-)
New file, header should say 2017 and OSGEO only.